src/ziggurat/streams.clj

```diff
@@ -31,6 +31,9 @@
    :oldest-processed-message-in-s      604800
    :changelog-topic-replication-factor 3
    :session-timeout-ms-config          60000
+   ;; :consumer-type                      :default
```

Comment:
why have we commented this out?

Reply:
are we getting this `:consumer-type` from config.edn or from the map that is passed to init-fn?

Reply:
what is the purpose of :consumer-type